lib/my_obfuscate/config_applicator.rb

```diff
@@ -80,6 +80,8 @@ def self.apply_table_config(row, table_config, columns)
             nil
           when :keep
             row[index]
+          when :ffaker
```

Comment:
Hey @mikeahmarani, How's this different from `type: :fixed, string: "hey there"`?